components/SignInForm.js

```diff
@@ -0,0 +1,38 @@
+import styles from "../styles/Home.module.css";
+import { Form, Row, Col } from "react-bootstrap";
+import { Button } from "react-bootstrap";
+import Chatrooms from "../pages/chatrooms";
+
+const SignIn = () => {
+  return (
+    <div className={styles.SignIn}>
+      <Form>
+        <Form.Group as={Row} className="mb-3" controlId="formPlaintextEmail">
+          <Form.Label column sm="2" className="BoldText">
+            Email
+          </Form.Label>
+          <Col sm="10">
+            <Form.Control
+              type="text"
+              plaintext
+              placeholder="email@example.com"
+            />
+          </Col>
+        </Form.Group>
+        <Form.Group as={Row} className="mb-3" controlId="formPlaintextPassword">
+          <Form.Label column sm="2">
+            Password
+          </Form.Label>
+          <Col sm="10">
+            <Form.Control type="password" placeholder="Password" />
+          </Col>
+        </Form.Group>
+        <Button variant="primary" href="/chatroom" onClick={Chatrooms}>
```

Comment:
you should be using a link to navigate to a page, I am not sure what the onclick accomplishes here. but it feels odd to import a page into a component.


```suggestion
        <Link href="/chatroom">
            <Button variant="primary">
                Sign In
            </Button>
        </Link>
```
the link component should be imported at the top

https://nextjs.org/docs/api-reference/next/link